From 1fb2f9fb2d75dfada602dbf35ee881ae6725ed84 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Tue, 19 Nov 2019 14:20:25 +0800
Subject: [PATCH 16/40] Revert "can: xilinx: fix RX FIFO overflow error
 handling"

This reverts commit 7c74ed7fbacecd1cbb55fb8bc7196d75239cd469.
---
 drivers/net/can/xilinx_can.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/can/xilinx_can.c b/drivers/net/can/xilinx_can.c
index 1f1c5b327301..5a24039733ef 100644
--- a/drivers/net/can/xilinx_can.c
+++ b/drivers/net/can/xilinx_can.c
@@ -670,7 +670,6 @@ static void xcan_update_error_state_after_rxtx(struct net_device *ndev)
 	}
 }
 
-static void xcan_chip_stop(struct net_device *ndev);
 /**
  * xcan_err_interrupt - error frame Isr
  * @ndev:	net_device pointer
@@ -720,8 +719,6 @@ static void xcan_err_interrupt(struct net_device *ndev, u32 isr)
 	if (isr & XCAN_IXR_RXOFLW_MASK) {
 		stats->rx_over_errors++;
 		stats->rx_errors++;
-		xcan_chip_stop(ndev);
-		xcan_chip_start(ndev);
 		if (skb) {
 			cf->can_id |= CAN_ERR_CRTL;
 			cf->data[1] |= CAN_ERR_CRTL_RX_OVERFLOW;
-- 
2.17.1

